src/legacy/status_im/test_runner.cljs

```diff
@@ -129,7 +129,8 @@
                            :transport/confirm-messages-processed
                            :group-chats/extract-membership-signature
                            :utils/dispatch-later
-                           :json-rpc/call})
+                           :json-rpc/call
+                           :fx})
```

Comment:
This needed to be added while running tests locally, otherwise the `rf/merge` function wouldn't return a merged collection of effects.